resources/tag_models/page.json

```diff
@@ -0,0 +1,57 @@
+{
+  "tag": {
+	"api_key": "", // String - API key - Make sure the request is from a valid client (idea nicked from Keen.io) useful if a page gets copied onto a Russian website and creates noise
+	"version": "", // Version of the tracking client e.g. "JS v1.2"
+	"type": "page", // Type of request (page or event)
+	"id": "", // ID of this request, must be unique
+	"offset": 0 // Delay of this tag between event happening and being sent to server - milliseconds
+  },
+
+  "page": {
+	"url": "",
+	"referrer": "",
+	"channel": "", // Our products e.g. ft.com, webapp, flipboard
```

Comment:
Yep, none of these are set in stone, just habit.

Rob Shilston just suggested we use cmdb name - although I think that'll end up being not the same definition.